07-Memory/kernel_space/out.log

```diff
@@ -0,0 +1,130 @@
+1, 0x3, 0.000000054, 0.000000050
+2, 0x4, 0.000000046, 0.000000049
+2, 0x5, 0.000000045, 0.000000050
+3, 0x8, 0.000000045, 0.000000049
+3, 0x9, 0.000000057, 0.000000052
+4, 0x10, 0.000000053, 0.000000049
+4, 0x11, 0.000000046, 0.000000049
+5, 0x20, 0.000000045, 0.000000049
+5, 0x21, 0.000000044, 0.000000050
+6, 0x40, 0.000000045, 0.000000049
+6, 0x41, 0.000000048, 0.000000049
+7, 0x80, 0.000000044, 0.000000051
+7, 0x81, 0.000000047, 0.000000056
+8, 0x100, 0.000000068, 0.000000082
+8, 0x101, 0.000000067, 0.000000092
+9, 0x200, 0.000000049, 0.000000053
+9, 0x201, 0.000000054, 0.000000055
+10, 0x400, 0.000000050, 0.000000057
+10, 0x401, 0.000000073, 0.000000065
+11, 0x800, 0.000000059, 0.000000060
+11, 0x801, 0.000000129, 0.000000087
+12, 0x1000, 0.000000079, 0.000000078
+12, 0x1001, 0.000000268, 0.000000127
+13, 0x2000, 0.000000118, 0.000000117
+13, 0x2001, 0.000000823, 0.000000278
+14, 0x4000, 0.000000278, 0.000000246
+14, 0x4001, 0.000001409, 0.000000514
+15, 0x8000, 0.000001886, 0.000000528
+15, 0x8001, 0.000003385, 0.000001483
+16, 0x10000, 0.000001636, 0.000001445
+16, 0x10001, 0.000006358, 0.000002910
+17, 0x20000, 0.000002636, 0.000002445
+17, 0x20001, 0.000005358, 0.000002310
+18, 0x40000, 0.000002636, 0.000002445
+18, 0x40001, 0.000006758, 0.000002610
+vmalloc
+iteration, size(hex), time, time free
+0, 0x1, 0.000004211, 0.000000303
+0, 0x2, 0.000000590, 0.000000210
+1, 0x2, 0.000001252, 0.000000262
+1, 0x3, 0.000001825, 0.000000228
+2, 0x4, 0.000000260, 0.000000199
+2, 0x5, 0.000000394, 0.000000213
+3, 0x8, 0.000000352, 0.000000212
+3, 0x9, 0.000000525, 0.000000190
+4, 0x10, 0.000000242, 0.000000203
+4, 0x11, 0.000000651, 0.000000222
+5, 0x20, 0.000000237, 0.000000204
+5, 0x21, 0.000000307, 0.000000210
+6, 0x40, 0.000000273, 0.000000227
+6, 0x41, 0.000000285, 0.000000212
+7, 0x80, 0.000000327, 0.000000198
+7, 0x81, 0.000000270, 0.000000195
+8, 0x100, 0.000000292, 0.000000198
+8, 0x101, 0.000000276, 0.000000209
+9, 0x200, 0.000000423, 0.000000194
+9, 0x201, 0.000000329, 0.000000219
+10, 0x400, 0.000000267, 0.000000206
+10, 0x401, 0.000000258, 0.000000206
+11, 0x800, 0.000000368, 0.000000218
+11, 0x801, 0.000000372, 0.000000218
+12, 0x1000, 0.000000280, 0.000000218
+12, 0x1001, 0.000000359, 0.000000299
+13, 0x2000, 0.000000514, 0.000000285
+13, 0x2001, 0.000001307, 0.000000388
+14, 0x4000, 0.000000530, 0.000000358
+14, 0x4001, 0.000004860, 0.000000480
+15, 0x8000, 0.000003066, 0.000000606
+15, 0x8001, 0.000003000, 0.000000587
+16, 0x10000, 0.000004592, 0.000001043
+16, 0x10001, 0.000004134, 0.000000902
+17, 0x20000, 0.000004892, 0.000001243
+17, 0x20001, 0.000004934, 0.000001002
+18, 0x40000, 0.000005592, 0.000001343
+18, 0x40001, 0.000005834, 0.000001602
+GFP
+iteration, size(hex), time, time free
+0, 0x1, 0.000000274, 0.000000135
+0, 0x2, 0.000000134, 0.000000069
+1, 0x2, 0.000000101, 0.000000079
+1, 0x3, 0.000000113, 0.000000093
+2, 0x4, 0.000000128, 0.000000105
+2, 0x5, 0.000000140, 0.000000128
+pt = NULL
```

Comment:
What is the maximum memory size allocated by get_free_pages?

Reply:
Allocated 2^5 pages

Reply:
Please check the correctness of this number.